boards/adafruit/metro_rp2350/adafruit_metro_rp2350_rp2350b_m33_defconfig

```diff
@@ -0,0 +1,14 @@
+# This configuration is orthogonal to whether the Cortex-M33 or Hazard3 cores
+# are in use, but Zephyr does not support providing a qualifier-agnostic
+# _defconfig file.
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_BUILD_OUTPUT_UF2=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_CONSOLE=y
+CONFIG_GPIO=y
+CONFIG_RESET=y
+CONFIG_SERIAL=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=150000000
```

Comment:
#90893 has been merged. Please rebase to latest, and remove this line.

Reply:
Done.